### PR TITLE
[conflict] Multiple worlds base

### PR DESCRIPTION
**Pull request by relinu**
Can add new worlds and switch back to old ones. No saving implemented atm and other worlds except the current world will not be updated. To change the world use World.ChangeWorld(World world).

### CODE REVIEW

**Comment by svmnotn** [comment]
please fix the conflicts

**Comment by vogonistic** [comment]
@relinu When you've fixed the conflicts, remove `[conflict]` from the title of this PR and we'll know to take a look at it.

**Comment by relinu** [comment]
I think I fixed everything, but this needs more testing

**Comment by Mizipzor** [comment]
Should we set it to [Wip] until testing happens?

**Comment by relinu** [comment]
@Mizipzor changed it

**Comment by RodRitter** [comment]
Conflict :(

**Comment by RodRitter** [comment]
@relinu Conflict :(

**Comment by relinu** [comment]
@RodRitter fixed

**Comment by RodRitter** [comment]
@relinu is this still a WIP? How can I test the world switching?

**Comment by relinu** [comment]
@RodRitter  Worldcontroller has newWorld and back bools set 'em to true, also World.ChangeWorld can be used

File: Assets/Scripts/Models/FurnitureActions.cs [hunk 1]
```diff
@@ -26,6 +26,11 @@ public FurnitureActions()
         LuaUtilities.RegisterGlobal(typeof(World));
     }
 
+    public static bool IsInitialized
+    {
+        get{ return _Instance != null; }
```
Comment (vogonistic) [inline Assets/Scripts/Models/FurnitureActions.cs]: The variable `_Instance` doesn't exist and the line should have the parenthesis on new lines.

**Comment by vogonistic** [comment]
Since so many of the callbacks are unregistered, this is mostly for switching between 2 worlds that are frozen when not active. Is that correct?

We really just need a better way to test it. Maybe always have a second world created and context click on open space to switch to it or something? Ping me on Discord and we can discuss it.

I'm flagging this PR `[conflict]`, even though it's a compiler error.

**Pull request closed without merging**